tests/MessagePack.SourceGenerator.Tests/Verifiers/CSharpSourceGeneratorVerifier`1+Test.cs

```diff
@@ -29,7 +29,7 @@ public static partial class CSharpSourceGeneratorVerifier
     public class Test : CSharpSourceGeneratorTest<EmptySourceGeneratorProvider, XUnitVerifier>
     {
         private readonly string? testFile;
-        private readonly string? testMethod;
+        private readonly string testMethod;
         private AnalyzerOptions options = AnalyzerOptions.Default;
 
         public Test([CallerFilePath] string? testFile = null, [CallerMemberName] string? testMethod = null)
```

Comment:
```suggestion
        public Test([CallerFilePath] string? testFile = null, [CallerMemberName] string testMethod = null!)
```

Reply:
That's a peculiar idiom, but it looks like it works. Thanks.